coalib/coala_json.py

```diff
@@ -28,12 +33,51 @@ def main():
     args = arg_parser.parse_args()
 
     log_printer = None if args.text_logs else ListLogPrinter()
+    JSONEncoder = create_json_encoder(use_relpath=args.relpath)
+
+    if args.show_bears:
+        try:
+            sections, _ = load_configuration(arg_list=None,
+                                             log_printer=log_printer)
+            local_bears, global_bears = collect_all_bears_from_sections(
+                sections, log_printer)
+            if args.filter_by_language:
+                local_bears = filter_section_bears_by_languages(
+                    local_bears, args.filter_by_language)
+                global_bears = filter_section_bears_by_languages(
+                    global_bears, args.filter_by_language)
```

Comment:
I'm planning to make a function out of line 40-48, but in which file should the function reside? `CliParsing.py`?